include/swift/Driver/DependencyGraph.h

```diff
@@ -121,6 +121,13 @@ class DependencyGraphImpl {
   llvm::StringMap<std::pair<std::vector<DependencyEntryTy>, DependencyMaskTy>> Dependencies;
 
   /// The set of marked nodes.
+  /// "Marked" means that everything provided by this node (i.e. Job) is
```

Comment:
Nitpick: Can you put an extra newline in here? We're formatting our comments as Doxygen (even if we never actually run the generator) and so the first "paragraph" is taken as a short summary.

Reply:
Done!